### ISSUE TITLE
Allied Base of Operations facility has enemy turrets?

### ISSUE THREAD

**Issue by SuperStucco** [MegaMek/mekhq#3713]
The StratCon facility definition file [AlliedBaseofOperations](https://github.com/MegaMek/mekhq/tree/master/MekHQ/data/stratconfacilities/AlliedBaseOfOperations.xml) has a scenario modifier entry like this:

<localModifiers>EnemyTurrets.xml</localModifiers>

Is this right?  The opposing facility, Hostile Base of Operations, has the EnemyTurrets modifier as well.  Should this be the AlliedTurrets scenario modifier for the allied facility?

**Comment by NickAragua** [comment]
Yeah, seems incorrect. Feel free to update, test (you can generate it on any contract by GM-mode adding a facility to an empty space) and submit a pull request.

**Comment by Thom293** [comment]
I noticed this a few days ago and already fixed it along with 2 other bugs.  I'm going on vacation this week but can push the changes in a week.

**Comment by Thom293** [comment]
Just got back in town. Will push this fix later this week.
